src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Based on this talk https://www.youtube.com/watch?v=VkTCL6Nqm6Y ~20min,  BEM actually makes a lot of sense besides personal style guide. Helps to resolve issues with class name conflicts and specificity clashes.